src/ui/info.ts

```diff
@@ -4,14 +4,16 @@
 
 async function info(_ui: UI) {
   const ui: UIConfig = {
-    data: {
-      info: '',
-      dropboxToken: localStorage.dropboxToken || '',
-      dropboxTokenUrl:
-          'https://www.dropbox.com/oauth2/authorize?response_type=token&client_id=013qun2m82h9jim&redirect_uri=' +
-          encodeURIComponent(chrome.extension.getURL('dropboxtoken.html'))
-    },
+    data: {info: '', dropboxToken: localStorage.dropboxToken || ''},
     methods: {
+      getDropboxToken: () => {
+        chrome.runtime.sendMessage({action: 'dropbox'});
+      },
+      logoutDropbox: async () => {
+        localStorage.removeItem('dropboxToken');
+        _ui.instance.dropboxToken = '';
+        _ui.instance.openLink('https://www.dropbox.com/account/connected_apps');
```

Comment:
I don't think we need to open page anymore since token is deleted from localstorage. It's not that important though, so you can merge if you think it should open this page.

Reply:
I think user may not logout dropbox frequently, and if they click the button, they may just want to disconnect the extension from dropbox. So open the page make sense.